packages/token-search-discovery-controller/CHANGELOG.md

```diff
@@ -7,19 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.0]
+
 ### Added
 
-- Introduce the `logoUrl` property to the `TokenSearchApiService` response
+- Rename `ControllerMessenger` to `Messenger` ([#5243](https://github.com/MetaMask/core/pull/5243))
```

Comment:
We can drop this item from the changelog. It is only an internal change and should not affect consumers:

``` suggestion
```